assets/app/view/game/spreadsheet.rb

```diff
@@ -156,6 +156,32 @@ def render_or_history_row(hist, corporation, x)
         end
       end
 
+      def render_connection_run(corporation)
+        return [] unless @game.respond_to?(:connection_run)
+        return [h(:td)] unless @game.connection_run[corporation]
+
+        x, c_run = @game.connection_run[corporation]
+        revenue_text, alpha =
+          case (c_run.dividend.is_a?(Engine::Action::Dividend) ? c_run.dividend.kind : 'withhold')
+          when 'withhold'
+            ["[#{c_run.revenue}]", '0.5']
+          else
+            [c_run.revenue.to_s, '1.0']
+          end
```

Comment:
Looks complicated. Wouldn’t `if c_run.dividend.kind == 'withhold'` do the same?

Reply:
It probably would. Most of this function was inspired by the render_or_history_row function. but 1870 doesn't have a half-pay, so I just removed that.
The connection runs also don't have many of the corner cases that or_history can run into, so it could probably be simplified.